plugin/src/main/kotlin/org/gradleweaver/plugins/jlink/JLinkExtension.kt

```diff
@@ -9,7 +10,39 @@ open class JLinkExtension
 internal constructor(
         project: Project
 ) {
-    val configure: NamedDomainObjectCollection<JLinkOptions> = project.container(JLinkOptions::class.java) { name ->
+    val configure: NamedDomainObjectContainer<JLinkOptions> = project.container(JLinkOptions::class.java) { name ->
```

Comment:
This needs to be renamed to avoid the name collision.
https://github.com/gradle/kotlin-dsl/issues/1175